react/ProductHighlightText.test.tsx

```diff
@@ -57,11 +57,7 @@ test("ProductHighlightText should render nothing if there's no context", () => {
 })
 
 test("ProductHighlightText should render nothing if there's no message", () => {
-  const { container } = render(
-    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-    // @ts-ignore
-    <ProductHighlightText />
-  )
+  const { container } = render(<ProductHighlightText message="{}" />)
```

Comment:
@victorhmp

Because of Lint's mistakes. In the old version it caused a Lint error

Reply:
Can you revert the changes in this file and see if the lint bot will complain here? It shouldn't do it. And if it does, it shouldn't affect your PR since you were not changing the tests initially.

Reply:
@icazevedo 

Corrections made.